src/clj/clojuredocs/pages/quickref/static.clj

```diff
@@ -258,14 +303,15 @@
            vals
            get
            get-in
+           update
            update-in
            select-keys
            merge
-           merge-with),
-         :title "Use (Sorted Maps)"}
+           merge-with
+           reduce-kv),
+         :title "Use"}
         {:syms (rseq subseq rsubseq),
-
-         :title "Use"})}
+         :title "Use (Sorted Maps)"})}
```

Comment:
It looks like these functions work on `Sorted` collections, not maps.

Reply:
I think you must have looked at one of the earlier commits. I was like that before, and I fixed it. :)

Reply:
Oh whoops, you're right. 

This would be a good time to fix it. Can you change `Sorted Maps` to `Sorted Collections`? Looking at the docstrings it seems like that would better represent the types of data structures these functions operate on.